specification/cognitiveservices/data-plane/CustomVision/Training/readme.md

```diff
@@ -77,3 +78,30 @@ python:
   basic-setup-py: true
   output-folder: $(python-sdks-folder)/azure-cognitiveservices-vision-customvision
 ```
+
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: training
+  clear-output-folder: true
+```
+
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: release_1_0
+```
+
+### Tag: release_1_0 and go
+
+These settings apply only when `--tag=release_1_0 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'release_1_0' && $(go)
+output-folder: $(go-sdk-folder)/services/cognitiveservices/v1.2/customvision/$(namespace)
```

Comment:
I wonder if we are doing the right thing with CS. Couldn't this be confusing ? It seems that the versioning is for each CS in part while in our sdk it seems its a version of CS itself.